templates/materializecssform/field.html

```diff
@@ -258,8 +258,9 @@
       {% endif %}
     </label>
     {% for value in field.value %}
-      <input type="date" id="{{ field.auto_id }}" class="datepicker col s5" name="{{ field.name }}_{{ forloop.counter0 }}"
-        value="{% if field.value %}{{ field.value|date:'Y-m-d' }}{% endif %}" data-value="{% if field.value %}{{ field.value|date:'Y-m-d' }}{% endif %}">
+      {% comment %} cycle after/before - this is assuming range will only iterate over 2 fields, required for django-filter instead of 0-1 indexes {% endcomment %}
+      <input type="date" id="{{ field.auto_id }}" class="datepicker col s5" name="{{ field.name }}_{% cycle 'after' 'before' %}"
```

Comment:
Just out of curiosity - have you tried to change these suffixes to ["0", "1"] in our widgets that override those from `django-filters`? It would be easier to control that from Python than from templates, but it's fine if that's the only way.

Reply:
That would work as well, the end result would be the same. Could make a base class that specifies the suffixes and the other widgets subclass it, but imo price_min in the url looks better than price_0 :P and holds more meaning

Reply:
True about the URLs. But all right, we can keep it in templates as well 👍